src/debugger/rnDebugSession.ts

```diff
@@ -49,6 +49,7 @@ export class RNDebugSession extends LoggingDebugSession {
 
     constructor(private session: vscode.DebugSession) {
         super();
+        this.isSettingsInitialized = false;
```

Comment:
Do we use that somewhere?

Reply:
Yes, `isSettingsInitialized` property is used [here](https://github.com/microsoft/vscode-react-native/blob/5bad552473b976a666370164cb5c210363ab2a2d/src/debugger/rnDebugSession.ts#L190) and [here](https://github.com/microsoft/vscode-react-native/blob/5bad552473b976a666370164cb5c210363ab2a2d/src/debugger/rnDebugSession.ts#L217) to prevent reinitialization of parameters in this [function](https://github.com/microsoft/vscode-react-native/blob/5bad552473b976a666370164cb5c210363ab2a2d/src/debugger/rnDebugSession.ts#L189).